packages/experimental/Spinner/src/Spinner.tsx

```diff
@@ -1,35 +1,26 @@
 /** @jsx withSlots */
-import { View } from 'react-native';
+import { Animated, View } from 'react-native';
 
 import type { UseSlots } from '@fluentui-react-native/framework';
 import { compose, mergeProps, withSlots } from '@fluentui-react-native/framework';
-import { TextV1 as Text } from '@fluentui-react-native/text';
 import { Svg } from 'react-native-svg';
 
-import { RCTNativeAnimatedSpinner } from './consts.win32';
 import type { SpinnerProps, SpinnerType } from './Spinner.types';
 import { spinnerName } from './Spinner.types';
 
-/* TODO: Implement Spinner with following slots */
+export const AnimatedSvg = Animated.createAnimatedComponent(Svg);
```

Comment:
>Animated.createAnimatedComponent [](http://example.com/codeflow?start=27&length=32)

ANimated doesn't work for win32, I thought?

Reply:
Yeah so I forked Spinner.win32 which doesn't use the Animated library. This file is being restored for other platforms

Reply:
Whoops my bad. Thanks